app/storage/encrypted_questionnaire_storage.py

```diff
@@ -42,14 +34,15 @@ def add_or_update(self, data, version):
         db.session.commit()
 
     def get_user_data(self):
+        data = None
         questionnaire_state = self._find_questionnaire_state()
         if questionnaire_state:
             data = json.loads(questionnaire_state.state)
             version = questionnaire_state.version or 0
 
-            if 'data' in data:
-                decrypted_data = self._decrypt_data(data['data'])
-                return decrypted_data, version
+        if data and 'data' in data:
```

Comment:
If we keep this nested inside the `if` above then we don't need the extra code here?